src/components/base/widget-position.js

```diff
@@ -315,6 +315,12 @@ export default WrappedComponent =>
 				ReactDOM.findDOMNode(this)
 			);
 
+			domElement.on('transitionend', () => {
+				domElement.setStyles({
+					pointerEvents: '',
+				});
+			});
+
```

Comment:
One thing to try before merging this solution: we might actually be able to avoid the whole "transitionend" thing if we change the `_animate()` implementation. I see it if doing an `if (window.requestAnimationFrame)` check and calling the `callback()` immediately if it is not present. I wonder if that explain the original behavior being different in IE (although "caniuse" *does* say that IE 11 has `requestAnimationFrame`). Changing that `else` to `setTimeout(callback, 0)` seems like the correct thing to do, even if it doesn't obviate the need for the "transitionend" trick.

In any case, I am happy to merge this as-is so I am hitting the "approve" button.